components/profile/layout/layout.tsx

```diff
@@ -17,11 +17,33 @@ const createRedirectCallback = (dashboardPath: string[]) => (appState: any) => {
   Router.replace(appState?.returnTo || dashboardPath.join("/"));
 };
 
+const shouldBypassAuth = (): boolean => {
+  if (typeof window === "undefined") return false;
+
+  const routesToBypass = ["/min-side/vipps-anonym"];
+  return routesToBypass.some((route) => window.location.pathname.startsWith(route));
+};
+
+const AuthenticatedLayout = ({ children }: { children: ReactNode }) => {
+  return (
+    <UserWrapper>
+      <DonorProvider>
+        <ActivityProvider>{children}</ActivityProvider>
+      </DonorProvider>
+    </UserWrapper>
+  );
+};
+
+const UnauthenticatedLayout = ({ children }: { children: ReactNode }) => {
+  return <>{children}</>;
+};
+
 export const ProfileLayout: React.FC<LayoutProps> = ({ children, footerData, widgetData }) => {
   const { dashboardPath } = useRouterContext();
   const [widgetOpen, setWidgetOpen] = useState(false);
   // Set true as default to prevent flashing on first render
   const [cookiesAccepted, setCookiesAccepted] = useState(true);
+  const bypassAuth = shouldBypassAuth();
```

Comment:
I agree that it may be a cleaner solution for this for code quality, but I think the current solution is fully functional and will achieve the desired behavior without problems. I am open to changing it in this PR if you think it would be beneficial though